inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/project/KnowledgeBaseDetailsPanel.html

```diff
@@ -106,6 +105,12 @@ <h3 wicket:id="name" class="panel-title"></h3>
         </div>
       </div>
       <div class="form-group">
+        <div>
+          <label class="col-sm-2 control-label">
+            <wicket:message key="language" />
+          </label>
+          <div wicket:id="language" class="col-sm-10"></div>
```

Comment:
I was not able to make the input field smaller during edit mode. Maybe you have an idea?